sites/website/README.md

```diff
@@ -1,161 +1,33 @@
-# What's in this document
+# Website
 
-This website was created with [Docusaurus](https://docusaurus.io/).
+This website is built using [Docusaurus 2](https://v2.docusaurus.io/), a modern static website generator.
 
-* [Get Started in 5 Minutes](#get-started-in-5-minutes)
-* [Directory Structure](#directory-structure)
-* [Editing Content](#editing-content)
-* [Adding Content](#adding-content)
-* [Full Documentation](#full-documentation)
+### Installation
 
-## Get started in 5 minutes
-
-1. Make sure all the dependencies for the website are installed:
-
-```sh
-npm install
-```
-
-2. Run your dev server:
-
-```sh
-npm start
 ```
-
-## Directory structure
-
-Your project file structure should look something like this
-
-```text
-fast-dna/
-  docs/
-    doc-1.md
-    doc-2.md
-    doc-3.md
-  website/
-    core/
-    node_modules/
-    pages/
-    static/
-      css/
-      img/
-    package.json
-    sidebar.json
-    siteConfig.js
+$ yarn
```

Comment:
Is this command specific to the `website/` folder, and if so - should we call that out?

Reply:
Ooops. Docusaurus generated that from their cli. I changed things around a bit and forgot about this file. I'll clean it up shortly.

Reply:
I fixed the readme to reflect the correct commands. Let me know if you think I should add more clarification. Most of this part was generated by the Docusaurus cli.